### PR TITLE
Fixes #1937 plus copy edits

### LINKED ISSUE #1937: Redis administration
The document in Redis Administration provides an incorrect command for disabling "transparent huge pages" on Linux Kernel.

It current states:
> ...This is accomplished with the following command: echo madvise > /sys/kernel/mm/transparent_hugepage/enabled.

The command should be
`echo never > /sys/kernel/mm/transparent_hugepage/enabled`

Source redhat: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/performance_tuning_guide/sect-red_hat_enterprise_linux-performance_tuning_guide-configuring_transparent_huge_pages

### PR DESCRIPTION
**Pull request by nermiller**

### CODE REVIEW

**Comment by nermiller** [comment]
@itamarhaber, please review when you can. Thanks! - Nermina

**Comment by nermiller** [comment]
@banker please review when you can. Thanks!

File: docs/manual/admin.md [hunk 1]
```diff
@@ -11,63 +11,78 @@ aliases: [
...
+`echo never > /sys/kernel/mm/transparent_hugepage/enabled`
 
-+ Use `daemonize no` when running under daemontools.
+### Memory
 
-+ Make sure to setup some non-trivial replication backlog, which must be set in proportion to the amount of memory Redis is using. In a 20 GB instance it does not make sense to have just 1 MB of backlog. The backlog will allow replicas to sync with the master instance much more easily.
+* Set up as much swap as memory in your system. If Linux does not have swap set up, and your Redis instance accidentally consumes too much memory, Redis can crash when it is out of memory, or the Linux kernel OOM killer can kill the Redis process. When swapping is enabled, you can detect latency spikes and act on them.
```
Comment (banker) [inline docs/manual/admin.md:28]:
```suggestion
* Ensured that swap is enabled and your swap file size is equal to amount of memory on your system. If Linux does not have swap set up, and your Redis instance accidentally consumes too much memory, Redis can crash when it is out of memory, or the Linux kernel OOM killer can kill the Redis process. When swapping is enabled, you can detect latency spikes and act on them.
```

**Review by banker — APPROVED** [review]

**Comment by banker** [comment]
Thanks @nermiller !

**Review by itamarhaber — COMMENTED** [review]
LOL - I was halfway reviewing this when it got merged in another timezone.

**Comment by nermiller** [comment]
@itamarhaber I'll wait for @yossigo's answer and open a new PR if needed.

**Comment by oranagra** [comment]
the change about THP is wrong IMHO: see https://github.com/redis/redis/pull/7771
maybe it could have mentioned that it should be `never` if you wanna silence warnings in redis versions prior to 6.2.

p.s. it is impossible to review this PR, it it includes both rephrasing and also actual content changes, it would have been better to split it into two (or even three, if some changes are just formatting and line splits)

**Pull request merged**